docs/modules/publisher-directmethods.md

```diff
@@ -65,46 +61,48 @@ The `_V1` direct methods  uses the  payload schema as described below:
-
-_Note_: `OpcNodes` field is mandatory for PublishNodes_V1, UnpublishNodes_V1 and AddOrUpdateEndpoints_V1. It should not be specified for the rest of the direct methods.
+| Attribute                           | Mandatory | Type    | Default                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| ----------------------------------- | --------- | ------- | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `EndpointUrl`                       | yes       | String  | N/A                       | The OPC UA server endpoint URL                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `UseSecurity`                       | no        | Boolean | false                     | Controls whether to use a secure OPC UA mode to establish a session to the OPC UA server endpoint                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
>false [](http://example.com/codeflow?start=62&length=5)

Has this changed over the versions? Formerly we where secure by design and we used a secure transport by default. Now we do not anymore?

Reply:
I don't think this changed lately, at least not from 2.7.x to 2.8.2. I believe the reason for this was that we wanted to be interoperable & easy to use on the first attempt first and then enable security. We can easily change this is required.